usaspending_api/api_contracts/contracts/v2/autocomplete/recipient.md

```diff
@@ -0,0 +1,55 @@
+FORMAT: 1A
+HOST: https://api.usaspending.gov
+
+# Recipient Autocomplete [/api/v2/autocomplete/recipient/]
+
+This endpoint is used by the Recipient autocomplete filter on the Advanced Search page.
+
+## POST
+
+This route sends a request to the backend to retrieve recipients matching the specified search text.
+
++ Request (application/json)
+    + Schema
+
+            {
+                "$schema": "http://json-schema.org/draft-04/schema#",
+                "type": "object"
+            }
+
+    + Attributes (object)
+        + `limit` (optional, number)
+            + Default: 10
+        + `search_text` (required, string)
+    + Body
+
+            {
+                "search_text": "Holdings"
```

Comment:
I have a similar question here to my questions about the Location filter autocomplete.  Per the mocks, there will be parenthesis in the search results on the frontend so just noting that the frontend team will need to know how the endpoint handles these so we know what to do with them on our end.  

Also, will the uei be searchable?

Otherwise, looks good from a frontend perspective.

Reply:
I'm going to push a small change to the contract to clarify that the backend endpoint will search for `search_text` in both the `recipient_name` and `uei` fields. Also that `recipient_levels` is a separate, optional, attribute that can also be supplied but isn't required.

As for the parenthesis, are you asking about how the endpoint would handle parenthesis if they were included in the `search_text`?